src/analysis/analyzer.ts

```diff
@@ -89,7 +89,7 @@ export class Analyzer extends AnalyzerGen implements vs.Disposable {
 			// This will include things like Observatory output and some analyzer logging code.
 			message = message.trim();
 			if (!message.startsWith('--- ') && !message.startsWith('+++ ')) {
-				logError({ message: `Unable to parse message (${e}): ${message}` });
+				console.error(`Unable to parse message (${e}): ${message}`);
```

Comment:
Checking the end for `}` would miss anything that might've been truncated (or our collecting of chunks gone bad?), but maybe worth doing startswith `{`. That said, I don't know how likely it is we'd get bad JSON back; I suspect the serialisation doesn't change much and that it'd be noticed well before a release if it was broken!